scripts/player/player.gd

```diff
@@ -0,0 +1,21 @@
+extends KinematicBody2D
+
+export (float) var speed := 200
+
+onready var animated_sprite := $AnimatedSprite
+
+var acceleration := 200
```

Comment:
This is not being used.

Reply:
Removed, I think the player acceleration wouldn't be used in this stage of the project.